api/censuses.go

```diff
@@ -157,8 +160,27 @@ func (capi *census3API) createAndPublishCensus(req *CreateCensusRequest, qID str
 	}
 	// check the censusType
 	censusType := defaultCensusType
+	finalAccuracy := 100.0
 	if req.Anonymous {
 		censusType = anonymousCensusType
+		// Round census balances using the default configuration
+		censusParticipants := []*roundedcensus.Participant{}
+		for address, balance := range strategyHolders {
+			censusParticipants = append(censusParticipants, &roundedcensus.Participant{
+				Address: address.String(),
+				Balance: balance,
+			})
+		}
+		res, accuracy, err := roundedcensus.GroupAndRoundCensus(censusParticipants, roundedcensus.DefaultGroupsConfig)
+		if err != nil {
+			log.Errorf("min accuracy couldn't be reached for strategy %d: %.2f: %v", req.StrategyID, accuracy, err)
```

Comment:
So if min accuracy is not reach, do we keep going?

Reply:
Yes, this is the current behaviour. The accuracy is stored with the census during the creation and can be estimated using the `/strategy/{id}/estimate` endpoint, so it is up to the user to decide if it is ok or not.